pkg/synopsysctl/cmd_create.go

```diff
@@ -283,10 +283,23 @@ func checkPasswords(flagset *pflag.FlagSet) {
 	}
 }
 
-func checkSealKey(flagset *pflag.FlagSet) {
+func verifyUserProvidedSealKey(flagset *pflag.FlagSet) {
```

Comment:
why we need function for it?

Reply:
I guess because it's used in 2 places (here and native)

Reply:
@msenmurugan Or are you asking - why don't we put this logic into ctl_blackduck.go ?

Reply:
we are not verifying anything in this method. so why do we need it?

Reply:
We don't check the value but we are requiring that the user set --seal-key for the create command.